desktop/apps/auction2/components/DOM.js

```diff
@@ -31,14 +31,14 @@ export default class DOM extends Component {
   addEventListeners () {
     this.$body = this.$('body')
     this.$body.on('click', '.artsy-checkbox', scrollToTop)
-    this.$(window).on('scroll.auction-page-artworks', infiniteScroll(this.props.store))
+    this.$(window).on('scroll.auction2-page-artworks', infiniteScroll(this.props.store))
```

Comment:
Yeah, global css state so things need to be namespaced to avoid conflicts. I'm sure this can be improved at some point but for now this is a simple quick fix during dev. Another example: https://github.com/artsy/force/pull/1591